src/index.ts

```diff
@@ -1,6 +1,3 @@
-// Ensure @import rules for fonts are before anything else. Otherwise build tools can show warnings
-// in consuming apps.
```

Comment:
Will the warnings come back?

Reply:
the warnings were happening b/c the mixin caused at-import statements to appear throughout the index.css file. By moving it to fonts.css, the warning won't return, b/c index.css won't try to import font files anymore